rclpy/src/rclpy/node.hpp

```diff
@@ -60,7 +60,7 @@ class Node : public Destroyable, public std::enable_shared_from_this<Node>
    * \return String containing the fully qualified name of the node
    */
   const char *
-  get_node_fully_qualified_name();
+  fully_qualified_name();
```

Comment:
@ahcorde nit: why `fully_qualified_name()` and `get_namespace()`? I mean, why not always putting `get_`? Or never?  Consistency would be best.